runtime/builtin_types.go

```diff
@@ -827,3 +861,18 @@ func zipLongest(f *Frame, args Args) ([][]*Object, *BaseException) {
 	}
 	return result, nil
 }
+
+func round(number float64, places int) float64 {
+	var round float64
```

Comment:
Just implemented it as `math.Floor(f + 0.5)`, and raises NotImplementError while passing second paramter on it. How about it?